packages/dx-react-grid-bootstrap3/src/templates/pager.jsx

```diff
@@ -22,7 +22,7 @@ export const Pager = ({
 
   return (
     <div
-      className={classNames('clearfix', className)}
+      className={classNames('clearfix', 'panel-footer', className)}
```

Comment:
It the `clearfix` class  is needed here?

Reply:
It's useful on small screens.